acra-censor/common/logging_logic.go

```diff
@@ -69,7 +69,7 @@ func NewFileQueryWriter(filePath string) (*QueryWriter, error) {
 	// read existing queries
 	err = writer.readStoredQueries()
 	if err != nil {
-		writer.logger.WithError(ErrCantReadQueriesFromFileError).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorIOError).Errorln("Can't create QueryWriter instance")
+		writer.logger.WithError(ErrCantReadQueriesFromStorageError).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorIOError).Errorln("Can't create QueryWriter instance")
```

Comment:
we should log origin error too

Reply:
It will be logged from readStoredQueries function